components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1010,8 +1010,7 @@ void updateConsumerApplication(OAuthConsumerAppDTO consumerAppDTO, boolean enabl
                 // 2. The AT claims separation is not enabled at app level.
                 // 3. The access token claims are empty.
```

Comment:
do we need this line

Reply:
Fixed with cf05de54bfdcb2f2d985791996eb0ea858fffce1